src/Tools/AI Test Toolkit/src/TestSuite/AITTestMethodLines.Page.al

```diff
@@ -0,0 +1,266 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.TestTools.AITestToolkit;
+
+page 149034 "AIT Test Method Lines"
+{
+    Caption = 'Tests';
+    PageType = ListPart;
+    SourceTable = "AIT Test Method Line";
+    AutoSplitKey = true;
+    DelayedInsert = true;
+    Extensible = false;
+
+    layout
+    {
+        area(content)
+        {
+            repeater(Control1)
+            {
+                field("LoadTestCode"; Rec."Test Suite Code")
+                {
+                    ToolTip = 'Specifies the ID of the AIT.';
```

Comment:
Move tooltips to record level (and I suggest replacing the AIT acronym with the full name)